dev-php/fedora-autoloader/fedora-autoloader-0.2.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Fedora's Static PSR-4, PSR-0, and classmap autoloader"
+HOMEPAGE="https://github.com/php-fedora/autoloader"
+
+# here we use autoloadder as it is the upstream name
```

Comment:
It was a choice I made with the help of @orlitzky 2 arguments for that:
* stick with the fedora name to avoid naming issue.
* stick to the namespace, that's the thing I've done in all other case, that's help finding the dependencies.
* extra it is a reference to fedora t have started the project which helped a lot doing that.

So I dropped the comment line, but keep the name, are you ok @mgorny  ?